deployment/agent/flyte_agent_helm_generated.yaml

```diff
@@ -71,6 +71,9 @@ spec:
         helm.sh/chart: flyteagent-v0.1.10
         app.kubernetes.io/managed-by: Helm
     spec:
+      securityContext:
+        seLinuxOptions:
+          type: spc_t
```

Comment:
I'm wondering why we need this?

Reply:
I think it's useful. I'd just make it configurable and disabled by default. We haven't seen much on-prem users running RHEL/Openshift but if this is useful for them, at least making it available would help.